tests/unit/test_utils.py

```diff
@@ -38,7 +38,7 @@ def test_build_vcs_link():
 
 def test_strip_ssh_from_git_url():
     url = "git+ssh://git@github.com/sarugaku/passa.git"
-    url_no_ssh = "git+git@github.com/sarugaku/passa.git"
```

Comment:
It looks like ```[-e] git+git://git.example.com/MyProject#egg=MyProject``` is also the supported format, should we support it and test on it?

Reply:
@jxltom Yeah, but the point here is to change `git+git@` to a standardized URL. And `git://` URL will be kept as it is. So as long as pip supports it, Pipenv should be, too.

Reply:
That sounds nice. We just need to make sure there is no regressions. It will be nice to populate the tests.

Reply:
Makes sense, trying to add more tests.